sdk/src/events/eventSubscriber.ts

```diff
@@ -57,8 +58,32 @@ export class EventSubscriber {
 				this.connection,
 				this.address,
 				this.options.commitment,
-				this.options.resubTimeoutMs
+				this.options.logProviderConfig.resubTimeoutMs
 			);
+			if (this.options.logProviderConfig.resubTimeoutMs) {
+				const logProviderConfig = this.options
+					.logProviderConfig as WebSocketLogProviderConfig;
+				this.logProvider.eventEmitter.on('reconnect', (reconnectAttempts) => {
```

Comment:
think this.logProvider needs to be logProviderConfig ?

Reply:
the event emitter is on the logp rovider